src/libcpu/src/jit/jit.cpp

```diff
@@ -35,6 +38,12 @@ sJitBlocks;
 static uint8_t
 sBaseRelocCode[16];
 
+static void *
+gVerifyPreWrapper;
```

Comment:
Will do. Also g -> s since they're static (I originally had them extern but changed to static later).